packages/graphql-language-service-types/src/index.ts

```diff
@@ -283,11 +293,24 @@ export type CustomValidationRule = (
 
 export type Diagnostic = DiagnosticType;
 
+export type CompletionItemBase = {
+  label: string;
+  isDeprecated?: boolean;
+};
+
 export type CompletionItem = CompletionItemType & {
   isDeprecated?: boolean;
   deprecationReason?: string;
 };
 
+export type CompletionItemForCodeMirror = {
```

Comment:
what is diffferent about this type? do you just need to make it `deprecationReason?: string | null`

Reply:
In [a successive PR](#902), `getAutocompleteSuggestions` for CodeMirror returns an object that only contains properties `label`, `type`, `documentation`, `isDeprecated`, `deprecationReason`. It doesn't contain many of properties of `CompletionItemType`, so I thought using a type designated for it, which is  `CompletionItemForCodeMirror`, it is easier to understand what `getAutocompleteSuggestions` for CodeMirror returns.